src/pharmpy/tools/common.py

```diff
@@ -90,7 +90,10 @@ def create_results(
         [base_model.modelfit_results] + [m.modelfit_results for m in res_models]
     )
 
-    best_model_name = summary_tool['rank'].idxmin()
+    try:
+        best_model_name = summary_tool['rank'].idxmin()
+    except FutureWarning:
```

Comment:
Is `FutureWarning` an exception being raised?

Reply:
That exception is raised if all models are filtered out because then `idxmin` raises an error.

Reply:
The integration tests are currently failing on Mac due to this warning happening in `test_amd` on the COVSearch step. One option could be to use the `skipna`-option for `idxmin()`, this would return `NA` if the whole column is `NA`, then we could set the best model to the input model if `NA` is returned.